src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -336,6 +336,15 @@ class GTGRunner(val masterURL: String,
     processEdges(MCCEdgeList, MCCNodeMap)
 
     /**
+     * Generate the netcdfs
+     */
+    val broadcastedNodeMap = sc.sparkContext.broadcast(MCCNodeMap)
+    val edgeTupleList = sc.sparkContext.parallelize(MCCEdgeList.toList).map({ x =>
```

Comment:
Kim,

You can avoid the extra parallelize by not collecting the rdd initially. Run the task on the rdd and then collect